site/content/docs/5.0/forms/form-control.md

```diff
@@ -35,15 +35,15 @@ Add the `disabled` boolean attribute on an input to give it a grayed out appeara
 
 {{< example >}}
 <input class="form-control" type="text" placeholder="Disabled input" aria-label="Disabled input example" disabled>
-<input class="form-control" type="text" placeholder="Disabled readonly input" aria-label="Disabled input example" disabled readonly>
+<input class="form-control" type="text" value="Disabled readonly input" aria-label="Disabled input example" disabled readonly>
 {{< /example >}}
 
 ## Readonly
 
-Add the `readonly` boolean attribute on an input to prevent modification of the input's value. Read-only inputs appear lighter (just like disabled inputs), but retain the standard cursor.
+Add the `readonly` boolean attribute on an input to prevent modification of the input's value. Read-only inputs appear bolder (just like disabled inputs), but retain the standard cursor.
```

Comment:
this isn't right is it? The font-weight doesn't change with the readonly attribute.